docs/pages/access-controls/guides/passwordless.mdx

```diff
@@ -219,6 +220,23 @@ $ tsh touchid diag
 # Touch ID enabled? true
 ```
 
+### Windows Webauthn not usable
+
+Windows Webauthn support for `tsh` requires Windows 10 19H1 or later.
+
+You can run the `tsh webauthnwin diag` command to verify requirements. A capable
+device and `tsh` binary should show an output similar to the one below:
```

Comment:
When this sentence says "similar", what variations in the output below are acceptable? Are there specific values that indicate that your environment is correct?

Reply:
we use similar because this is like debug command and we could add some stuff later without updating docs.

Reply:
If the output of `tsh webauthnwin diag` _does_ differ from the docs, how would a user be able to tell that the output is sufficiently similar to the example below? Is there a specific field they should look for? Are there error messages if the command is not successful?

Reply:
i have removed similar wording and reworked it a little bit. I will update docs with example if we change diag output.